curriculum/challenges/english/08-coding-interview-prep/rosetta-code/semiprime.english.md

```diff
@@ -0,0 +1,116 @@
+---
+id: 5eb3e4aa847216613aa81983
+title: Semiprime
+challengeType: 5
+forumTopicId: 385318
+---
+
+## Description
+
+<section id='description'>
+Semiprime numbers are natural numbers that are products of exactly two (possibly equal) <a href="https://rosettacode.org/wiki/prime_number" target="_blank">prime numbers</a>.
+<pre>1679  =  23 x 73</pre>
+</section>
+
+## Instructions
+
+<section id='instructions'>
+Write a function that returns true if a number is semiprime, or false if it is not.
```

Comment:
```suggestion
Write a function that returns true if a number is semiprime, and false if it is not.
```

Reply:
This could go either way. It seems like our current challenge use both "and" or "or" for this type of sentence.